integration/spark/src/main/scala/org/apache/spark/sql/execution/strategy/DDLHelper.scala

```diff
@@ -52,15 +54,16 @@ object DDLHelper {
     ThreadLocalSessionInfo
       .setConfigurationToCurrentThread(sparkSession.sessionState.newHadoopConf())
     if (!EnvHelper.isLegacy(sparkSession)) {
-      val databaseName = createDatabaseCommand.databaseName
-      val dbLocation = try {
-        CarbonEnv.getDatabaseLocation(databaseName, sparkSession)
-      } catch {
-        case _: NoSuchDatabaseException =>
-          CarbonProperties.getStorePath
+      val carbonStorePath = CarbonProperties.getStorePath()
+      val sparkWarehouse = sparkSession.conf.get("spark.sql.warehouse.dir")
+      if (StringUtils.isNotEmpty(carbonStorePath) && StringUtils.isNotEmpty(sparkWarehouse)
```

Comment:
do we need to fail it, when user fires create database with location ? i think, i that case, no need to check this

Reply:
I have modified code according to your suggestion